elastic/integrations
# [security-external-integrations packages] Update to ECS 8.5

## What does this PR do?

This updates the ECS version used in all non-deprecated packages owned by elastic/security-external-integrations.

See #4338 for a list of preparatory changes that were made before this PR.

## Checklist


## How to ~test~ recreate this PR locally

The changeset was fully produced by this command from the root of the repo at the parent of the first commit in the PR:

```
go run github.com/andrewkroh/go-examples/ecs-update@6efa1ecb3871 \
	-ecs-version=8.5.0 \
	-ecs-git-ref=v8.5.1 \
	-pr=4285 \
	-owner=elastic/security-external-integrations \
	packages/*
```

Note that the marked ECS version has a zero patch version as this appears to be the convention.

## Related issues

- Relates #4338
